fleet/lib/ccs-grafana/fleet.yaml

```diff
@@ -0,0 +1,19 @@
+---
+defaultNamespace: ccs-monitor
+labels:
+  bundle: ccs-grafana
+kustomize:
+  dir: base
+helm:
+  chart: &chart grafana
+  releaseName: *chart
+  repo: https://grafana.github.io/helm-charts
+  version: 7.3.11
+  timeoutSeconds: 600
```

Comment:
Does this chart take 10 minutes to deploy?

Reply:
I got it down to 5mins as I check some bundles have 300 secs, but yeah, 10 mins were overkill xD